media/CfgEditor/cfgeditor.js

```diff
@@ -0,0 +1,326 @@
+  panelCodegenDisabled.style.display = 'block';
+}
+
+function enableProfileStep() {
+  const profileEnabled = document.getElementById('profileEnabled');
+  const profileDisabled = document.getElementById('profileDisabled');
+  const panelProfileEnabled = document.getElementById('panelProfileEnabled');
+  const panelProfileDisabled = document.getElementById('panelProfileDisabled');
+
+  profileEnabled.style.display = 'block';
+  profileDisabled.style.display = 'none';
+  panelProfileEnabled.style.display = 'block';
+  panelProfileDisabled.style.display = 'none';
+
+  clickUnfoldAdvencedOptions();
+  const advancedOptions = document.getElementById('advancedOptions');
+  advancedOptions.activeid = 'tabProfile';
+}
+
+function disableProfileStep() {
+  const profileEnabled = document.getElementById('profileEnabled');
+  const profileDisabled = document.getElementById('profileDisabled');
+  const panelProfileEnabled = document.getElementById('panelProfileEnabled');
+  const panelProfileDisabled = document.getElementById('panelProfileDisabled');
+
+  profileEnabled.style.display = 'none';
+  profileDisabled.style.display = 'block';
+  panelProfileEnabled.style.display = 'none';
+  panelProfileDisabled.style.display = 'block';
+}
+
+function modelTypeClick() {
+  const modelType = document.getElementById('modelTypeRadio').value;
+  const initialImportState = document.getElementById('ImportInitialState');
+  const tfConverterOptions = document.getElementById('TFConverterOptions');
+  const tflConverterOptions = document.getElementById('TFLITEConverterOptions');
+  const onnxConverterOptions = document.getElementById('ONNXConverterOptions');
+
+  initialImportState.style.display = 'none';
+  tfConverterOptions.style.display = 'none';
+  tflConverterOptions.style.display = 'none';
+  onnxConverterOptions.style.display = 'none';
+
+  switch (modelType) {
+    case 'pb':
+    case 'savedModel':
+    case 'kerasModel':
+      tfConverterOptions.style.display = 'block';
+      return;
+    case 'tflite':
+      tflConverterOptions.style.display = 'block';
+      return;
+    case 'onnx':
+      onnxConverterOptions.style.display = 'block';
+      return;
+    default:
+      return;
+  }
+}
+
+function clickUnfoldAdvencedOptions() {
+  const foldButton = document.getElementById('foldAdvancedOptions');
+  const unfoldButton = document.getElementById('unfoldAdvancedOptions');
+  const advancedOptions = document.getElementById('advancedOptions');
+
+  advancedOptions.style.display = 'block';
+  foldButton.style.display = 'inline';
+  unfoldButton.style.display = 'none';
+}
+
+function clickFoldAdvancedOptions() {
+  const foldButton = document.getElementById('foldAdvancedOptions');
+  const unfoldButton = document.getElementById('unfoldAdvancedOptions');
+  const advancedOptions = document.getElementById('advancedOptions');
+
+  advancedOptions.style.display = 'none';
+  foldButton.style.display = 'none';
+  unfoldButton.style.display = 'inline';
```

Comment:
optional) these patterns are error prone, I guess.
Maybe in future, we could define some function like this 😸 :

`toggleDispalyStyle( [ advancedOptions, foldButton, unfoldButton ] );`